tests/common/mod.rs

```diff
@@ -14,6 +14,7 @@ pub use runner::{assert_execution_result, mock_account, test_execute_alloy, test
 pub mod storage;
 
 pub type ChainState = AHashMap<Address, EvmAccount>;
+pub type Bytecodes = AHashMap<B256, EvmCode>;
```

Comment:
We can even move this to and use it in `InMemoryStorage`.

Reply:
`AHashMap<B256, EvmCode>` is used in various places. For example, in `src/storage/rpc.rs`, we have `cache_bytecodes: Mutex<AHashMap<B256, EvmCode>>`.

Should we move the type definition to `src/lib.rs`?

Reply:
Yeah, let's do it.

Reply:
Ah, we can put it in `storage` where `EvmCode` and the friends are. Not the best place but we can move everything together later.

Reply:
Fixed in the latest commit.